### PR TITLE
Fix ugly causatives panel on Case page

### LINKED ISSUE #3953: Causative variants window looks ugly on case page
Check it in comparison with the other 2:

image

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3953

Without causatives or pinned:

image

With Causatives and pinned:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 1]
```diff
@@ -396,7 +398,7 @@ <h6><span class="fa fa-medkit ms-3 mt-2"></span>
           <div class="list-group-item list-group-item-action">
             {% if variant._id %}
               {% do already_displayed_variant_ids.append(variant._id) %}
-              <div class="row align-items-between align-items-center">
+              <div class="d-flex align-items-between align-items-center">
```
Comment (dnil) [inline scout/server/blueprints/cases/templates/cases/utils.html:401]: Same here, but there Im less sure we really want justify-content-between?

**Review by dnil — APPROVED** [review]
Aligned and nice! Thanks for the explanation. Oh, and don't forget to either drop the align-items-between or change to justify-content-between. I think the latter looks slightly nicer to the eye, but it would be consistent enough either way.

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 2]
```diff
@@ -347,17 +347,19 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
 {% macro causatives_list(causatives, partial_causatives, evaluated_variants, institute, case, manual_rank_options, cancer_tier_options) %}
   {% set already_displayed_variant_ids = [] %}
   <div class="card mt-3">
-    <div data-bs-toggle='tooltip' title="Displays all variants that have been marked causative for this case">
-      <div class="d-flex justify-content-between">
+     <div data-bs-toggle='tooltip' title="Displays all variants that have been marked causative for this case">
+      <div class="d-flex align-items-between align-items-center">
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/utils.html:351]: The become a bit ugly, check it out:

image

[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/utils.html:351]: Then you need to add spacing also on the right, which is doable, but I find that like it is works anyway

[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/utils.html:351]: It's as before, minus the row context spacing/padding I suppose.

[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/blueprints/cases/templates/cases/utils.html:351]: All good, just remove it! There is no such thing as `align-items-between`. 😸

[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/cases/templates/cases/utils.html]: Right! :)

Looks good after removing them!

image

**Pull request merged**